src/MeetingsSDKAdapter.js

```diff
@@ -57,6 +57,7 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     super(datasource);
     this.getMeetingObservables = {};
     this.meetings = {};
+    this.audioEnabled = true;
```

Comment:
This cannot be a instance variable. The meetings adapter acts like a singleton manager for potentially multiple meetings.

Reply:
I think I found a better solution opening up a new PR closing this out